internal/k8s/loader/load.go

```diff
@@ -59,11 +59,7 @@ func FetchClusterLogForwarder(k8sClient client.Client, namespace, name, logstore
 	extras := map[string]bool{}
 	forwarder.Spec, extras = migrations.MigrateClusterLogForwarderSpec(forwarder.Namespace, forwarder.Name, forwarder.Spec, fetchClusterLogging().Spec.LogStore, extras, logstoreSecretName)
 
-	if fetchClusterLogging().Name == "" {
-		extras[constants.ClusterLoggingAvailable] = false
-	} else {
-		extras[constants.ClusterLoggingAvailable] = true
-	}
+	extras[constants.ClusterLoggingAvailable] = (fetchClusterLogging().Name != "")
```

Comment:
nit: redundant parentheses

Reply:
Thought it was clearer that way, just look at it without the parens:
```
extras[constants.ClusterLoggingAvailable] = fetchClusterLogging().Name != ""
```